.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: ZGrab2 build action
+on: [push, pull_request]
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - name: Set up Docker for integration tests
+        uses: docker-practice/actions-setup-docker@master
+
+      - name: Check out source
+        uses: actions/checkout@v2
+
+      - name: Install goimports
+        run: sudo apt install golang-golang-x-tools
+
+      - name: Build
+        run: |
+          go get -t ./...
+          make
+
+      - name: Test
+        run: make clean zgrab2 test
+        continue-on-error: true
+
+      - name: Integration test
+        run: make integration-test
+        continue-on-error: true
+
+      - name: Integration cleanup
+        run: ./integration_tests/cleanup.sh
```

Comment:
I think we can just drop the integration tests from CI. I'm pretty sure we broke them when we switched to Go modules. Did you have any luck getting them to run locally?

Reply:
No, I didn't successfully get this running locally. If you or someone else has a better understanding of how it should work, that might help with troubleshooting. I thought about reverting the repo to the last commit where TravisCI successfully passed and checking if the tests worked locally for me then, but I didn't go that far.

Since I did not get these tests working, I could remove my edits of the runner-base.Dockerfile file. I only modified it while trying to get it working and it's possible my edits, while minor, might move the file further from a functioning state.

Reply:
I'd leave the Dockerfile alone, and drop the integration tests from CI. Can you file a follow up issue to figure out:

1. What they did
2. How to run them now that Go modules exist?

@justinbastress might have insights, he wrote them originally, back in the dark days of Go development.